rdr_service/offline/main.py

```diff
@@ -795,12 +767,18 @@ def export_va_workqueue_report():
     export_va_workqueue.generate_workqueue_report()
     return '{"success": "true"}'
 
+
 @app_util.auth_required_cron
 def delete_old_va_workqueue_reports():
     export_va_workqueue.delete_old_reports()
     return '{"success": "true"}'
 
 
+@app_util.auth_required_cron
+def aw3_ready_missing_files_report():
```

Comment:
Need to add the decorator here for enabling/disabling this job.

Reply:
The decorator has been added.